kong/runloop/balancer.lua

```diff
@@ -746,13 +763,20 @@ local function init()
   end
   log(DEBUG, "initialized ", oks, " balancer(s), ", errs, " error(s)")
 
+  local upstream_update_frequency = kong.configuration.upstream_update_frequency or 1
+  timer_every(upstream_update_frequency, function(premature)
```

Comment:
This timer is not needed when the `upstream_consistency` is `"strict"`, as the upstreams must be kept always updated.